__tests__/melt-tokens.test.js

```diff
@@ -0,0 +1,67 @@
+import TestUtils from "./test-utils";
+
+// TODO: Add a fixture to melt token 01
+describe("melt-tokens api", () => {
+  it("should return 200 with a valid body", async () => {
+    const response = await TestUtils.request
+      .post("/wallet/melt-tokens")
+      .send({
+        token: "00da712d64e04866c8c9aa8fceca70e80d1693864176b6b443220cf29adab5ed",
+        amount: 1,
+      })
+      .set({ "x-wallet-id": TestUtils.walletId });
+    expect(response.status).toBe(200);
+    expect(response.body.hash).toBeDefined();
```

Comment:
I think we should validate the return output has only a melt authority. Not the deposit because you've melted 0.01

Reply:
Testing it with a mocked`fullnode` is not a valid effort for now.